api/src/test/java/org/pmiops/workbench/billing/FreeTierBillingServiceTest.java

```diff
@@ -936,6 +933,7 @@ private void assertSingleWorkspaceTestDbState(
   private DbUser createUser(String email) {
     DbUser user = new DbUser();
     user.setUsername(email);
+    user.setFirstRegistrationCompletionTime(Timestamp.from(START_INSTANT));
```

Comment:
since we now only check if the user has a valid registration time

Reply:
aside: We ought to revisit the user data model. We could have a separate registration table that holds . the start, end, repeat count, etc. This is a couple of degrees removed from the actual user's core state.